grayskull/pypi/pypi.py

```diff
@@ -552,11 +560,23 @@ def _get_metadata(self) -> dict:
         version = ""
         if self["package"]["version"].values:
             version = self.get_var_content(self["package"]["version"].values[0])
-        pypi_metadata = self._get_pypi_metadata(name, version)
-        sdist_metadata = self._get_sdist_metadata(
-            sdist_url=pypi_metadata["sdist_url"], name=name
-        )
-        metadata = PyPi._merge_pypi_sdist_metadata(pypi_metadata, sdist_metadata)
+
+        #print({name})
+
+        if name.startswith(("http://", "https://")):
```

Comment:
How come we need this logic again? Didn't we test for these prefixes already above?

Reply:
This is the only place we are checking this

Reply:
In pypi.py, that is